### PR TITLE
Jhalackova removing butane

### PR DESCRIPTION
**Pull request by lvicoun**
### Description

Describe the overall goals of this pull request.

### Are there any relevant issues/feature requests?

* bsc#...
* jsc#SLE-...

### Is this (based on) existing content?

* old xml:id...
* old URL...

### CODE REVIEW

File: tasks/deployment-adding-users.xml [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+ xmlns:xlink="http://www.w3.org/1999/xlink"
+ xmlns:trans="http://docbook.org/ns/transclusion">
+  <info>
+    <title>Adding users</title><!-- can be changed via merge
+in the assembly -->
+    <!-- add author's e-mail -->
+    <meta name="maintainer" content="jsindelarova@suse.com" its:translate="no"/>
+    <abstract><!-- can be changed via merge in the assembly -->
+      <para>
+        As &productname; requires to have a non-privileged user to login via SSH or to access  &cockpit;, you need to create such an account.
```
Comment (dariavladykina) [inline tasks/deployment-adding-users.xml:27]:
```suggestion
        As &productname; requires having a non-privileged user to log in via SSH or to access &cockpit;, you need to create such an account.
```

File: tasks/deployment-adding-users.xml [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+      </para>
+    </abstract>
+  </info>
+  <para>
+    This step is optional if you defined an unprivileged user in the &ignition; or
+    &combustion;. If you deployed your system using &jeosfirstboot;, then you set up only the
+    &rootuser; password and you need to create the unprivileged account manually as described bellow:
```
Comment (dariavladykina) [inline tasks/deployment-adding-users.xml:34]:
```suggestion
    This step is optional if you have defined an unprivileged user in the &ignition; or
    &combustion;. If you deployed your system using &jeosfirstboot;, then you set up only the
    &rootuser; password and you need to create the unprivileged account manually, as described below:
```

File: references/ignition-configuration.xml [hunk 3]
```diff
@@ -23,11 +23,7 @@
     <title>Configuration examples</title>
     <para>
       This section provides several examples of the &ignition; configuration in
-      both the built-in JSON format and the YAML format as well. &ignition; does
-      not accept configuration in the YAML format, and you need
-      to convert it to the JSON format. To do so, you can use the
-      <literal>butane</literal> tool as described in
-      <xref linkend="deployment-convert-yaml-to-json"/>.
+       the built-in JSON format and the YAML format
```
Comment (dariavladykina) [inline references/ignition-configuration.xml:26]:
```suggestion
       the built-in JSON format and the YAML format.
```

**Review by dariavladykina — COMMENTED** [review]
LGTM, just some minor things. Thanks!

File: tasks/deployment-adding-users.xml [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+    </step>
+    <step>
+      <para>
+        Set a password for that account:
+      </para>
+      <screen>&prompt.root;passwd <replaceable>USER_NAME</replaceable></screen>
+    </step>
+    <step>
+      <para>
+        If needed, add the user to the <literal>wheel</literal> group:
```
Comment (tbazant) [inline tasks/deployment-adding-users.xml:51]: i had the feeling that the `wheel` group is being discourages in favor of `sudo`?

**Review by tbazant — APPROVED** [review]
looks good, just a small nit

**Pull request merged**